frontend-react/src/hooks/UseValueSets.ts

```diff
@@ -131,13 +132,14 @@ export const useValueSetUpdate = () => {
 
     // generic signature is defined here https://github.com/TanStack/query/blob/4690b585722d2b71d9b87a81cb139062d3e05c9c/packages/react-query/src/useMutation.ts#L66
     // <type of data returned, type of error returned, type of variables passed to mutate fn, type of context (?)>
-    const mutation = useMutation<LookupTable, Error, UpdateValueSetOptions>(
-        updateValueSet
-    );
+    const mutation = useMutation<
+        LookupTable,
+        RSNetworkError,
```

Comment:
nice - I think this needs to be updated in the activation call on line 159 as well

Reply:
Good catch!